src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final Random random = new Random();
+    private static final int ballCount = 100;
+    
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(ballCount) + 1);
```

Comment:
Why do we need this +1?
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(ballCount));
```

Reply:
'coz nextInt returning values not include ballCount and we need that 100 as max value